iron-list.html

```diff
@@ -1349,16 +1349,26 @@
         this._scrollHeight = this._estScrollHeight;
       }
     },
+
     /**
      * Scroll to a specific item in the virtual list regardless
      * of the physical items in the DOM tree.
      *
-     * @method scrollToIndex
-     * @param {number} idx The index of the item
+     * @method scrollToItem
+     * @param {(Object|number)} item The item object or its index
      */
-    scrollToIndex: function(idx) {
-      if (typeof idx !== 'number') {
-        return;
+    scrollToItem: function(item){
+      var idx = null;

```

Comment:
I was thinking that scrollToIndex would eventually be deprecated since scrollToItem can take either an object or an index....